### PR TITLE
initial-commit

### PR DESCRIPTION
**Pull request by SaadMahi**
Balmain shoes Product UI Card!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <div class="container">
+      <div class="card">
+        <div class="imgBx">
+          <img
+            src="https://int.balmain.com/dw/image/v2/BHGX_PRD/on/demandware.static/-/Sites-balmain-mastercatalog/default/dwebdae5a8/images/hi-res/AM1VJ309TNCSEGUF.png?sw=1680&q=80"
+            alt=""
```
Comment (murtazshaikh) [inline index.html]: always add values in alt attribute

**Review by murtazshaikh — APPROVED** [review]
Adding a favicon was a good idea.
LGTM!

File: index.html (hunk 1 above)
Comment (SaadMahi) [reply index.html]: value added!

**Review by OSAMA478 — COMMENTED** [review]
group similar properties ...

**Comment by SaadMahi** [comment]
> group similar properties ...

similar properties grouped as suggested.

**Review by ManSOorcode — APPROVED** [review]
👌

**Pull request merged**